### DISCUSSION TITLE
Kopia based backup

### DISCUSSION THREAD

**Discussion by weshayutin** [backube/volsync#474] [Ideas]
Kopia is replacing restic as the default backup tool used in velero.  This change will take place in Velero 1.11 
Restic will still be available in Velero through a deprecation cycle, however Kopia may offer some advantages and performance improvements.  It's not 100% clear yet and needs discussion but I thought it would be worth noting here.

Thank you

Here is some information:
Velero team’s original investigation report comparing performance of restic and kopia [Copy of kopia-restic-comparison.docx - Google Docs](https://docs.google.com/document/d/1ghc8vvj8q0w0lJ0RD5vm_lSeRbt3x3ho/edit)
Epic: [Integrate with Kopia for pod volume backup/restore · Issue #5050 · vmware-tanzu/velero (github.com)](https://github.com/vmware-tanzu/velero/issues/5050)
Design Doc [velero/unified-repo-and-kopia-integration.md at main · vmware-tanzu/velero (github.com)](https://github.com/vmware-tanzu/velero/blob/main/design/unified-repo-and-kopia-integration/unified-repo-and-kopia-integration.md)

**Comment by JohnStrunk** [comment]
Thanks for the pointer to that comparison document. It provides a good overview of the trade-offs.
